sectionsUtilityTool/src/main/webapp/error.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Error page for Sections Manager</title>
+  </head>
+  <body class="container-fluid" style="padding-top:20px;">    
+  	<h1>Acccess denied!</h1>
```

Comment:
Color me fussy.  A 403 means that you aren't permitted access and authenticating won't help. 401 means that you aren't authenticated and should be.  The text and the status code are inconsistent.